app/graphql/types/auth_provider_credentials_input.rb

```diff
@@ -0,0 +1,8 @@
+module Types
+  class AuthProviderCredentialsInput < BaseInputObject
+    graphql_name 'AUTH_PROVIDER_CREDENTIALS'
+
+    argument :username, String, required: true
+    argument :password, String, required: true
+  end
+end
```

Comment:
I think this file should  be moved to `types/input/auth_provider_credential_input.rb` to follow the rest of the code

Reply:
You are right. I'll make that change now and commit directly.